mmv1/products/healthcare/terraform.yaml

```diff
@@ -60,7 +60,9 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           pubsub_topic: "fhir-notifications"
           bq_dataset_name: "bq_example_dataset"
         test_vars_overrides:
-          policyChanged: 'BootstrapPSARoles(t, "gcp-sa-healthcare", []string{"roles/bigquery.dataEditor", "roles/bigquery.jobUser"})'
+          policyChanged: 'BootstrapPSARoles(t, "service-", "gcp-sa-healthcare",
+
+            []string{"roles/bigquery.dataEditor", "roles/bigquery.jobUser"})'
```

Comment:
Nit: Was this changed because of the line length? I think I'd advocate for keeping it on one long line vs. the extra newlines.

Another possible option (I'm not an expert on the options with yaml):
```
policyChanged: |
  BootstrapPSARoles(t, "gcp-sa-healthcare", []string{"roles/bigquery.dataEditor", "roles/bigquery.jobUser"})
```

Reply:
I wanted the generated code to put the function call on two lines after formatting and this seemed like the best way to accomplish that.

Reply:
Apparently breaking up the lines with `|` breaks the build, but wrapping it in `'`s is fine.